cosmwasm/devnet/config/config.toml

```diff
@@ -370,7 +370,7 @@ timeout_precommit_delta = "500ms"
 # How long we wait after committing a block, before starting on the new
 # height (this gives us a chance to receive some more precommits, even
 # though we already have +2/3).
-timeout_commit = "0.5s"
+timeout_commit = "1s"
```

Comment:
Why are we making this change?  I don't see anything in the commit or PR description that explains the reason.

Reply:
This change is required to sync terra2 and wormchain using IBC. A `timeout_commit` of 0.5s is too fast and leads to Terra2's clock going into the future. Once the clocks on each chain differ by > 10 minutes, we can no longer sync IBC headers.

Reply:
Thanks for the explanation.  This is useful context that should be documented somewhere, either in the comments in this config or in the commit message.